Fakebook.Posts/Fakebook.Posts.IntegrationTests/ClientActions/PostCreation.cs

```diff
@@ -248,5 +252,32 @@ public async Task PostAsyncComment_InvalidPost_BadRequest()
             Assert.Equal(HttpStatusCode.BadRequest, response.StatusCode);
         }
 
+
+
+        [Fact]
+        public void GetNewsFeedAsyn_DtoIsNotNull_ReturnsOk()
+        {
+
+                        // Arrange
+            Mock<IPostsRepository> mockRepo = new();
+            Mock<IFollowsRepository> mockFollowRepo = new();
+            Mock<IBlobService> mockBlobService = new();
+            Mock<Microsoft.Extensions.Logging.ILogger<Fakebook.Posts.RestApi.Controllers.PostsController>> mockLoggerService = new();
```

Comment:
use using directives, not fully qualified names. and fix the formatting

Reply:
used using directive